tests/functional/test_submitted_charts.py

```diff
@@ -224,6 +224,11 @@ def submission_tests_run_for_submitted_charts(secrets):
             chart_dir = f'charts/{vendor_type}/{vendor_name}/{chart_name}'
             base_branch = f'{secrets.pr_base_branch}-{vendor_type}-{vendor_name}-{chart_name}-{chart_version}'
             pr_branch = f'{secrets.pr_base_branch}-{vendor_type}-{vendor_name}-{chart_name}-{chart_version}-pr'
+
+            if not secrets.software_name:
```

Comment:
Confused you use secrets.software_name is it is not defined?

Reply:
This will check is empty or None. Either case yeah what is the point on using below

Reply:
Actually this is redundant, somehow I was thinking during manual trigger the `software-name` is optional. Updating